website/client/components/shops/shopItem.vue

```diff
@@ -29,15 +29,14 @@ div
     slot(name="popoverContent", :item="item")
       equipmentAttributesPopover(
         v-if="item.purchaseType==='gear'",
-        :item="item"
-      )
+        :item="item")
       div.questPopover(v-else-if="item.purchaseType === 'quests'")
         h4.popover-content-title {{ item.text }}
         questInfo(:quest="item")
       div(v-else)
-        h4.popover-content-title(v-once) {{ item.text }}
-        .popover-content-text(v-if="showNotes", v-once) {{ item.notes }}
-
+        h4.popover-content-title(v-once) {{ item.text }} {{item.key}}
```

Comment:
do two different {{ }} block work? I thought only was permitted

Reply:
It works, but that is not intended. It was meant for debugging.